pkg/api/workflow/v1/deepcopy_generated.go

```diff
@@ -1,20 +1,6 @@
 // +build !ignore_autogenerated
 
-/*
-Copyright 2017 The Kubernetes Authors.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
+// Generated file, do not modify manually!
```

Comment:
The license has been removed. The different generators take in argument the boilerplate template for the generated file  `--go-header-file  hack/boilerplate.go.txt` . But I think you forget to commit it.

Reply:
ah , ok thanks.